### PR TITLE
Start examples

### PR DESCRIPTION
**Pull request by jzohdi**
In this PR I added a simple nextjs project to `examples/`.

I also added a couple ignore rules into flake as it should not add px to `lineHeight`, `flex`.

### CODE REVIEW

**Comment by samouri** [comment]
Please update the package.json `main` to point to `index.min.js`

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "@sightread/flake",
-  "version": "0.5.5",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "version": "0.5.9",
```
Comment (samouri) [inline package.json]: no bump necessary

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  "main": "./dist/index.min.js",
+  "typings": "./dist/index.d.ts",
   "license": "MIT",
   "files": [
     "dist/"
   ],
   "scripts": {
-    "build": "tsc && terser dist/index.js -c -m -o dist/index.min.js && gzip dist/index.min.js -k -f",
+    "build": "tsc && terser dist/index.js -c -m -o dist/index.min.mjs && gzip dist/index.min.js -k -f",
```
Comment (samouri) [inline package.json]: this needs to stay `.js`

**Review by samouri — CHANGES_REQUESTED** [review]

**Review by samouri — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jzohdi) [reply package.json]: fixed

File: package.json (hunk 2 above)
Comment (jzohdi) [reply package.json]: returned back to `.js`

**Pull request merged**